dash_table/index.html

```diff
@@ -5,9 +5,8 @@
     </head>
     <body>
         <div id='root'></div>
-        
-        <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
-        <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+        <script src='https://unpkg.com/react@16.8.6/umd/react.production.min.js'></script>
+        <script src='https://unpkg.com/react-dom@16.8.6/umd/react-dom.production.min.js'></script>
```

Comment:
Slighty tighter versioning -- we know Dash targets a specific version of React, no need to add potential instability through testing with the latest.

Reply:
Is there any (easy) way to tie this explicitly to the version in dash? Or perhaps just a test that they're the same, so if we change it in dash we'll be prompted to update it here?

Reply:
Interesting. Nothing I can think up right now. Opening a follow up story.

Reply:
https://github.com/plotly/dash-table/issues/522